docs/statistics.md

```diff
@@ -195,37 +223,29 @@ The total number of packets that failed to be decrypted at the receiver side. Av
 
 ### pktSndFilterExtraTotal
 
-The total number of packet filter control packets supplied by the packet filter (refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Available for sender.
+The total number of packet filter control packets sent by the SRT sender (refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Available for sender.
 
-Packet filter control packets are SRT DATA packets.
+The type of the packet filter control packets is DATA. These packets contain only packet filter control information and do not belong to the actual transmitted data.
 
 The `SRTO_PACKETFILTER` socket option should be enabled (refer to [API.md](API.md)). Introduced in SRT v1.4.0.
```

Comment:
I'd rather say that if the packet filter hasn't been enabled by `SRTO_PACKETFILTER` option, this will be always 0.

Reply:
It's not true. If the option is not enabled, there would be no statistic present at all, at least in the csv output. I'll stay with my version.